src/Tribe/Service_Providers/Context.php

```diff
@@ -270,6 +270,38 @@ static function ( $post_type ) {
 					Tribe__Context::REQUEST_VAR => [ TEC::TAXONOMY ],
 				],
 			],
+			'view_url' => [
+				'read'  => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'read'" and double arrow, but found 2.

<sub>hash: 82aaa00d5d8bc58a72c729660f25fc98</sub>